ext/bin/run-load-test

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+from sys import exit, stderr, stdout
+from datetime import datetime
+import os, sys, getopt, subprocess, shutil
+
+def usage(stream):
+    usage = """
+Usage: run-load-test --host http://myhost:port [options]
+
+Options:
+  -h [--host]       Host to which the requests are made
+  -u [--users]      Number of users to simulate. Default 1
+  -s [--spawn_rate] The rate per second in which users are spawned. Default 1
+  -t [--timelimit]  How long the swarm should take. Default 1m (Examples: 30s, 1h30m)
+  -p [--prefix]     After running csv and html report files are created, use this option to add a custom prefix to the files. Default datetime formated as MMDDYYYY-HHMM
+  --help            Show this help"""
+    print(usage, file=stream)
+
+def misuse():
+    usage(stderr)
+    exit(2)
+
+options, remainder = getopt.getopt(sys.argv[1:], 'h:u:s:t:p:', ['host=',
+                                                            'users=',
+                                                            'spawn_rate=',
+                                                            'timelimit=',
+                                                            'prefix=',
+                                                            'help'
+                                                       ])
+
+users = 1
+spawn_rate = 1
+timelimit = "1m"
+now = datetime.now()
+prefix = now.strftime("%m%d%Y-%H%M")
+
+for opt, arg in options:
+    if opt in ('-h', '--host'):
+        host = arg
+    elif opt in ('-u', '--users'):
+        users = arg or users
+    elif opt in ('-s', '--spawn_rate'):
+        spawn_rate = arg or users
+    elif opt in ('-t', '--timelimit'):
+        timelimit = arg or timelimit
+    elif opt in ('-p', '--prefix'):
+        prefix = arg or prefix
+    elif opt == '--help':
+        usage(stdout)
+        exit()
+    else:
+        usage(stdout)
+        exit(2)
+
+if 'host' not in locals():
+  usage(stdout)
+  exit(2)
+
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+workdir = os.getcwd()
+subprocess.check_call([sys.executable, "-m", "pip", "install", "locust", "pyyaml", "-U"])
+subprocess.run(["locust", "-f", dir_path + "/load-test/load-test.py", "--headless", "-H", host, "-u", str(users), "-r", str(spawn_rate), "-t", timelimit, "--csv", prefix, "--html", prefix + "_report.html"])
+
+report_files = ['_exceptions.csv', '_failures.csv', '_stats_history.csv', '_stats.csv', '_report.html']
+for file in report_files:
+    filename = prefix + file
+    source = workdir + '/' + filename
+    destination = dir_path + '/load-test/output/'
+    shutil.move(source, destination)
```

Comment:
if the files already exist, shutil.move will throw an exception. I think that something like 
```
from pathlib import Path    
if Path(destination).is_file:
        os.remove(destination)
```
might work. Not sure about permissions.

Reply:
Updated to overwrite the files if they exist.